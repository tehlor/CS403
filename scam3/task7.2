(define x 0)
(sdisplay 5 (mystery x)) (println)
(sdisplay 5 (ps-mystery x)) (println)
(sdisplay 5 (acc-mystery x)) (println)
(sdisplay 5 (super-mystery x)) (println)
(symbolic-mystery)
