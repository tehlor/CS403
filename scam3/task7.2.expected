(1.0000000000,-0.000000e+00,0.000000e+00,-0.000000e+00,0.000000e+00,...)
(1.0000000000,1.0000000000,1.0000000000,1.0000000000,1.0000000000,...)
(1.0000000000,1.0000000000,1.0000000000,1.0000000000,1.0000000000,...)
(1.0000000000,1.0000000000,1.0000000000,1.0000000000,1.0000000000,...)
(mystery x) is $\cos x$
